pkg/ebpf/config.go

```diff
@@ -0,0 +1,134 @@
+package ebpf
+
+import (
+	"github.com/aquasecurity/libbpfgo/helpers"
+
+	"github.com/aquasecurity/tracee/pkg/containers/runtime"
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/events/queue"
+	"github.com/aquasecurity/tracee/pkg/pcaps"
+	"github.com/aquasecurity/tracee/pkg/policy"
+	"github.com/aquasecurity/tracee/pkg/signatures/engine"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// Config is a struct containing user defined configuration of tracee
+type Config struct {
```

Comment:
Nice! We have issue #3031 to extract config package out of the ebpf package, and I think this takes us almost there